dotcom-rendering/src/server/lib/logging.ts

```diff
@@ -29,6 +29,7 @@ const logFields = (logEvent: LoggingEvent): unknown => {
 		level_value: logEvent.level.level,
 		request,
 		fastlyRequestId,
+		abTests,
```

Comment:
Is this an object and therefore needs to be stringified before being logged?

Reply:
Yes, you're quite right! I've stringified this 👍